src/core/system.js

```diff
@@ -33,6 +33,16 @@ System.prototype = {
    * @param {number} timeDelta - Difference in current render time and previous render time.
    */
   tick: undefined,
+    
+  /**
+   * Post handler.
+   * Called after each iteration of the scene render loop.
+   * Affected by play and pause.
+   *
+   * @param {number} time - Scene tick time.
+   * @param {number} timeDelta - Time it took to render the scene.
+   */
+  tick: undefined,

```

Comment:
We probably shouldn't be using the `system` API to handle post-processing. Systems are sort of global managers for components. We should just use the plain component API.